pkg/backends/influxdb/flux/errors.go

```diff
@@ -0,0 +1,33 @@
+package flux
+
+import "fmt"
+
+type ErrFluxSyntax struct {
+	token string
+	rule  string
+}
+
+func FluxSyntax(token, rule string) *ErrFluxSyntax {
```

Comment:
... and _this_ should be called ErrFluxSyntax since it provides the error Object  :-D